ssn/ssn_separated/ssn.ttl

```diff
@@ -670,9 +658,9 @@ ssn:Observation rdf:type owl:Class ;
                                   owl:minCardinality "0"^^xsd:nonNegativeInteger
                                 ] ,
                                 [ rdf:type owl:Restriction ;
-                                  owl:onProperty ssn:featureOfInterest ;
+                                  owl:onProperty sosa:featureOfInterest ;
```

Comment:
the oldssn.ttl is a copy of the separated/ssn.ttl, that is progressively emptied whenever some terms are added into ssnx.ttl, sosa.ttl, ssn.ttl.
this is described in the steps 1 and 2 of the SOSA/SSN Integration Methodology [1,2]

[1] - https://www.w3.org/2015/spatial/wiki/OldSSN/SOSA/SSN_Integration_Methodology 
[2] - https://github.com/w3c/sdw/blob/gh-pages/ssn/integrated/readme.md